wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -138,6 +162,16 @@ public static function collectionCampTabset($tabsetName, &$tabs, $context) {
     ];
 
     foreach ($tabConfigs as $key => $config) {
+      // Skip the 'monetaryContribution' tab for 'goonj_chapter_admin'.
+      if ($skipMonetaryContributionTab && $key === 'monetaryContribution') {
+        continue;
+      }
+
+      // Skip specific tabs for 'account_team'.
+      if ($skipAccountTeamTabs && in_array($key, $accountTeamExcludedTabs)) {
+        continue;
+      }
```

Comment:
```suggestion
      // Skip if the current user does not have the required permissions.
      $hasPermission = \CRM_Core_Permission::check($config['permissions']);
      if (!hasPermission) {
        continue;
      }
```

Reply:
To make this change you will have to change the `$tabConfig` variable slightly like this:

```php
$tabConfigs = [
    //...

    'campFeedback' => [
        'title' => ts('Volunteer Feedback'),
        'module' => 'afsearchVolunteerFeedback',
        'directive' => 'afsearch-volunteer-feedback',
        'template' => 'CRM/Goonjcustom/Tabs/CollectionCamp.tpl',
        'permissions' => [], // add permissions required to view the tab.
    ],
    'monetaryContribution' => [
        'title' => ts('Monetary Contribution'),
        'module' => 'afsearchMonetaryContribution',
        'directive' => 'afsearch-monetary-contribution',
        'template' => 'CRM/Goonjcustom/Tabs/CollectionCamp.tpl',
        'permissions' => [], // add permissions required to view the tab.
    ],
];
```